pkg/scheduler/partition.go

```diff
@@ -1496,6 +1547,28 @@ func (pc *PartitionContext) removeAllocation(release *si.AllocationRelease) ([]*
 	return released, confirmed
 }
 
+func (pc *PartitionContext) removeForeignAllocation(allocID string) {
+	nodeID := pc.foreignAllocs[allocID]
```

Comment:
Do we really need all these warning cases?

Reply:
Might not be necessary... perhaps convert them to DEBUG?

Reply:
Yeah, DEBUG seems good.